python/prereq_parse.py

```diff
@@ -0,0 +1,180 @@
+from langchain.chat_models import ChatOpenAI
+from langchain import PromptTemplate, LLMChain
+from langchain.output_parsers import ResponseSchema
+from langchain.output_parsers import StructuredOutputParser
+from langchain.chains import SequentialChain, TransformChain
+
+OPEN_AI_API_KEY = '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'
+MODEL = 'gpt-3.5-turbo'
+verbose = False
+llm = ChatOpenAI(openai_api_key=OPEN_AI_API_KEY, temperature=0, model=MODEL)
+
+
+def get_raw_prereqs_and_coreqs(verbose=False):
+    '''
+    Returns a chain that takes in a course description and returns the prerequisites and corequisites as two separate lists.
+    '''
+    template =\
+    """
+    Return the course names that are required prerequisites for this class as list of strings. Do the same for corequisites.
+    Ignore any recommendations for prerequisites or corequisites.
+    A course name consists of an abbreviation in all capital letters and a 4 digit number.
```

Comment:
In my opinion, no because the reason for adding those there is to help it provide the correct response, but if it already does that with no examples, I don't really see the point in doing so because it just adds to the cost. I can definitely add a comment showing sample input and output though.